spec/shared_stripe_examples/payment_method_examples.rb

```diff
@@ -128,6 +128,17 @@
         expect { payment_method }.to raise_error(Stripe::InvalidRequestError)
       end
     end
+
+    context 'when a payment_method is supplied' do
+      let(:type) { nil }
+      let(:original) { Stripe::PaymentMethod.create(type: 'card') }
+      let(:payment_method) { Stripe::PaymentMethod.create(payment_method: original.id, type: type) }
```

Comment:
In Stripe, you can do `payment_method.card` and will work, but not here since we didn't pass the type 😞

Reply:
Is there anything that you'd like me to change before it's ready to merge?